File: clients/typescript/tests/workspaces/java/src/main/java/com/nuanced/FizzBuzz.java
package com.nuanced;

public final class FizzBuzz {
  public static String fizzBuzz(int n) {
    if (n % 15 == 0) return "fizzbuzz";
    if (n % 3 == 0) return "fizz";
    if (n % 5 == 0) return "buzz";
    return Integer.toString(n);
  }
}

File: clients/typescript/tests/workspaces/php/src/FizzBuzz.php
<?php
namespace Nuanced;

final class FizzBuzz {
  public static function fizzbuzz(int $n): string {
    if ($n % 15 === 0) return "fizzbuzz";
    if ($n % 3 === 0) return "fizz";
    if ($n % 5 === 0) return "buzz";
    return strval($n);
  }
}
